src/databaseTest/java/net/sf/jabref/shared/DBMSSynchronizerTest.java

```diff
@@ -43,15 +42,14 @@
     @Before
     public void setUp() throws ClassNotFoundException, SQLException, DatabaseNotSupportedException {
 
-        Globals.prefs = JabRefPreferences.getInstance();
-
         connection = TestConnector.getTestConnection(dbmsType);
 
         bibDatabase = new BibDatabase();
         BibDatabaseContext context = new BibDatabaseContext(bibDatabase);
 
 
-        dbmsSynchronizer = new DBMSSynchronizer(context);
+        dbmsSynchronizer = new DBMSSynchronizer(context,
+                JabRefPreferences.getInstance().get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
Somehow it doesn't matter since the keyword groups totally ignores the given keyword separator... #1877